src/aiortc/rtcrtpsender.py

```diff
@@ -175,6 +176,14 @@ def replaceTrack(self, track: Optional[MediaStreamTrack]) -> None:
         else:
             self._track_id = str(uuid.uuid4())
 
+    @property
+    def enabled(self) -> bool:
+        return self.__enabled
+
+    @enabled.setter
+    def enabled(self, enabled: bool) -> None:
+        self.__enabled = enabled
```

Comment:
Same as for `RTCRtpReceiver`, this property does not exist on in the JS interface. If it's just for our internal use, maybe just use a `_enabled` (single underscore).